back/app/services/notifications/internal_comments/internal_comment_on_initiative_assigned_to_you_builder.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module Notifications
+  class InternalComments::InternalCommentOnInitiativeAssignedToYouBuilder < InternalComments::BaseNotificationBuilder
+    protected
+
+    def skip_recipient?(recipient)
+      initiator_id.nil? ||
+        initiator_id == recipient.id ||
+        recipient.id == parent_author_id ||
+        MentionService.new.user_mentioned?(internal_comment.body, recipient)
+    end
+
+    def recipients
+      [User.find(internal_comment&.post&.assignee_id)]
```

Comment:
I proposed it, but now I realize that in some cases (like this and similar ones) it will raise an error. E.g., if the user is searched by `nil`.
```
[19] pry(main)> User.find(nil)
ActiveRecord::RecordNotFound: Couldn't find User without an ID
```
But it can be easily fixed like that
```suggestion
      [User.find_by(id: internal_comment&.post&.assignee_id)].compact
```

Reply:
I should know that too. Thanks!